### PR TITLE
Wiki API Call unittests completed

### PR DESCRIPTION
**Pull request by Joeroble**

### CODE REVIEW

File: test_api_wiki_call.py [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+class TestAPI_Wiki_call(TestCase):
+
+    @patch('requests.Response.json')
+    def test_api_wiki_call_data_returned(self, mock_wiki_json):
+        mock_user_date = 'October 29'
+        example_wiki_api_response = ['October 29', ['October 29'], [''], ['https://en.wikipedia.org/wiki/October_29']]
+        mock_wiki_json.side_effect = [example_wiki_api_response]
+        wiki_api_response = API_Wiki_Call.API_Wiki_Call(mock_user_date)
+        self.assertTrue(mock_wiki_json, wiki_api_response.data)
```
Comment (claraj) [inline test_api_wiki_call.py:15]: Should this be an assertEqual?

assertTrue has an optional second argument, a message to print if the test fails. 
https://docs.python.org/3/library/unittest.html#unittest.TestCase.assertTrue
So your code is checking if mock_wiki_json is true, or truthy - in Python, anything that's not 0, empty string/list/dictionary or False; is true, so mock_wiki_json is truthy; then the wiki_api_response.data would be interpreted as a message to print if the test fails.

File: test_api_wiki_call.py [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+    @patch('requests.Response.json')
+    def test_api_wiki_call_user_error(self, mock_wiki_jason):
+        mock_user_error = 'completegibberishtogetusererror'
+        example_wiki_api_response = [mock_user_error, [],[],[]]
+        mock_wiki_jason.side_effect = [example_wiki_api_response]
+        wiki_api_response = API_Wiki_Call.API_Wiki_Call(mock_user_error)
+        self.assertTrue(mock_wiki_jason, wiki_api_response.user_error)
```
Comment (claraj) [inline test_api_wiki_call.py:23]: Same as previous comment

File: test_api_wiki_call.py (hunk 1 above)
Comment (Joeroble) [reply test_api_wiki_call.py:15]: It should be, I do not know what I was even doing last night.  I wasn't checking what the actual result was in the api_response properly, which is silly of me.  I fixed both of these.

File: test_api_wiki_call.py (hunk 2 above)
Comment (Joeroble) [reply test_api_wiki_call.py:23]: I fixed this as well, I'm currently finishing it up.

**Review by aknayr — APPROVED** [review]

File: test_api_wiki_call.py (hunk 1 above)
Comment (claraj) [reply test_api_wiki_call.py]: Not silly, more that Python will let you make these mistakes more easily than a strongly-typed language like Java, Kotlin, C#. This type of error is somewhat common in Python so another reason why code reviews are so important.

**Pull request merged**